test/audio/PlaybackQueue-test.ts

```diff
@@ -0,0 +1,254 @@
+import { PlaybackQueue } from '../../src/audio/PlaybackQueue';
+import { MatrixClientPeg } from '../../src/MatrixClientPeg';
+import { mkEvent, stubClient } from '../test-utils';
+import { UPDATE_EVENT } from '../../src/stores/AsyncStore';
+
+jest.mock('../../src/audio/compat', () => ({
+    createAudioContext: jest.fn(),
+    decodeOgg: jest.fn(),
+}));
+
+describe('PlaybackQueue', () => {
+    stubClient();
+    const cli = MatrixClientPeg.get();
+    const buffer = new ArrayBuffer(8);
+    const waveform = null;
+
+    const updateCli = (events: MatrixEvent[], roomId: string): void => {
+        cli.getRoom = () => ({
+            roomId: roomId,
+            getLiveTimeline: () => ({
+                getEvents: () => events,
+            } as unknown as EventTimeline),
+        } as unknown as Room);
+    };
+
+    const mockAudioBufferSourceNode = {
+        addEventListener: jest.fn(),
+        connect: jest.fn(),
+        start: jest.fn(),
+    };
+    const mockAudioContext = {
+        decodeAudioData: jest.fn(),
+        suspend: jest.fn(),
+        resume: jest.fn(),
+        createBufferSource: jest.fn().mockReturnValue(mockAudioBufferSourceNode),
+    };
+
+    const mockAudioBuffer = {
+        duration: 99,
+        getChannelData: jest.fn(),
+    };
+
+    const mockChannelData = new Float32Array();
+
+    beforeEach(() => {
+        jest.spyOn(logger, 'error').mockRestore();
+        mockAudioBuffer.getChannelData.mockClear().mockReturnValue(mockChannelData);
+        mockAudioContext.decodeAudioData.mockReset().mockImplementation(
+            (_b, callback) => callback(mockAudioBuffer),
+        );
+        mockAudioContext.resume.mockClear().mockResolvedValue(undefined);
+        mockAudioContext.suspend.mockClear().mockResolvedValue(undefined);
+        mocked(decodeOgg).mockClear().mockResolvedValue(new ArrayBuffer(1));
+        mocked(createAudioContext).mockReturnValue(mockAudioContext as unknown as AudioContext);
+    });
+
+    it('two consecutive voice messages are played', async () => {
+        const roomIdTest = '!room0:server';
+        const ev0 = mkEvent({
+            event: true,
+            type: "m.room.message",
+            user: "@user1:server",
+            room: roomIdTest,
+            content: {
+                "org.matrix.msc2516.voice": {},
+            },
+        });
+
+        const ev1 = mkEvent({
+            event: true,
+            type: "m.room.message",
+            user: "@user1:server",
+            room: roomIdTest,
+            content: {
+                "org.matrix.msc2516.voice": {},
+            },
+        });
+
+        updateCli([ev0, ev1], roomIdTest);
+
+        const playback0 = PlaybackManager.instance.createPlaybackInstance(buffer, waveform);
+        const playback1 = PlaybackManager.instance.createPlaybackInstance(buffer, waveform);
+        const PlaybackQueueInstance = PlaybackQueue.forRoom(ev0.getRoomId());
```

Comment:
```suggestion
        const playbackQueueInstance = PlaybackQueue.forRoom(ev0.getRoomId());
```

Looks like it should be ^

Reply:
You can also use `roomIdTest` which would be clearer, IMO

Reply:
(both apply to all other cases of this)